riptide-failsafe/src/test/java/org/zalando/riptide/failsafe/CheckedPredicateConverterTest.java

```diff
@@ -0,0 +1,21 @@
+package org.zalando.riptide.failsafe;
+
+import dev.failsafe.function.CheckedPredicate;
+import org.junit.jupiter.api.Test;
+
+import java.util.function.Predicate;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
As in the other PR, I don't know in how this project typically expands `*` imports.